donor-registry/src/app/forms/forms.component.ts

```diff
@@ -1787,8 +1790,8 @@ export class FormsComponent implements OnInit {
     });
   }
   modalSuccess() {
-    var modal = document.getElementById("confirmationModal");
-    var btn = document.getElementById("submitBtn");
+    var modal = document.getElementById("downloadCardModalPledge");
```

Comment:
@chaitrali-r Have checked that this changes does not make the impact on livedonor/recipient?